Makefile

```diff
@@ -217,6 +217,9 @@ docker-clean-job:
 registry:
 	./utils/setup-registry.sh
 
+registry-down:
+	docker rm -f kind-registry || true
+
```

Comment:
So Racetrack is a system for bringing down registries? Neat!

Reply:
I realized that after doing all Quickstart steps, racetrack leaves this remnant registry, so I wanted to clean this up.

I'll rename this container to `racetrack-registry`, so people know it's associated with the racetrack's needs and there will be no risk of us destroying someone else's container.

Reply:
I was actually being cheeky and I meant that this doesn't seem related to the issue this PR is aimed at.